app/src/stores/post.spec.ts

```diff
@@ -263,30 +223,15 @@ describe("Post store", () => {
                 {
                     ...mockPost.content[0],
                     title: "post1",
-                    publishDate: now + 2000,
-                    expiryDate: now + 1000,
-                },
-            ],
-        } as any;
-        const post2 = {
-            ...mockPost,
-            _id: "2",
-            content: [
-                {
-                    ...mockPost.content[0],
-                    title: "post2",
-                    publishDate: now - 3000,
+                    publishDate: now - 2000,
                     expiryDate: now - 1000,
                 },
             ],
         } as any;
 
-        postStore.posts = [post1, post2];
+        postStore.posts = [post1];
 
-        const visiblePosts = postStore.posts.filter((post) => {
-            const { publishDate, expiryDate } = post.content[0];
-            return publishDate < now && (!expiryDate || expiryDate < now);
-        });
+        const visiblePosts = postStore.postsByTag(mockCategory._id);
 
         expect(visiblePosts.length).toBe(1);
         expect(visiblePosts[0].content[0].publishDate).toBeLessThan(now);
```

Comment:
The result of this test should be 0 (as the filter should have excluded the post). So this test is not testing a valid condition.